### PR TITLE
Showing my orders; approving and reimbursing

### PR DESCRIPTION
**Pull request by vanruch**

### CODE REVIEW

File: src/components/RootComponent.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import store from '../redux/stores/store';
 import ModalContainer from './stateless/specific/containers/ModalContainer/ModalContainer';
 import { BrowserHistory } from 'react-history';
 import CreateRequirements from '../components/hoc/CreateRequirementsHOC';
-import Product from './hoc/ProductHOC/ProductHOC';
+import Product from './hoc/ProductHOC/OrderSummaryHOC';
```
Comment (PunkSage) [inline src/components/RootComponent.jsx]: Please always use the same names for the class and the file containing it

Comment (marekkirejczyk) [reply src/components/RootComponent.jsx]: folder*

File: src/components/hoc/ProductContainerHOC.js [hunk 2]
```diff
@@ -26,18 +26,23 @@ const mapStateToProps = state => ({
 });
 
 const mapDispatchToProps = (dispatch) => ({
-  onMount: (address, qualities) => {
+  onMount: (address) => {
     if (address) {
-      getData(dispatch, address, qualities);
+      getData(dispatch, address);
```
Comment (PunkSage) [inline src/components/hoc/ProductContainerHOC.js]: Name is too generic, what kind of data are we getting?

File: src/components/hoc/ProductContainerHOC.js [hunk 3]
```diff
@@ -26,18 +26,23 @@ const mapStateToProps = state => ({
...
       let addressFromCookies = Cookies.get('market_address', address);
       if (addressFromCookies) {
         dispatch(gotoMarket({ address: addressFromCookies }));
-        getData(dispatch, addressFromCookies, qualities);
+        getData(dispatch, addressFromCookies);
       }
     }
   },
 
+  childrenData: (offer) => [
```
Comment (PunkSage) [inline src/components/hoc/ProductContainerHOC.js]: Name is too generic, who are children for this component?

File: src/components/RootComponent.jsx (hunk 1 above)
Comment (PunkSage) [reply src/components/RootComponent.jsx]: Folder as well, but I think that all HOC could be in one folder at the moment.

**Pull request merged**